edu/jalc/inclass/cocacola/manufacturing/SodaBottleFillerTest.java

```diff
@@ -1,21 +1,34 @@
 package edu.jalc.inclass.cocacola.manufacturing;
 
+import edu.jalc.inclass.cocacola.material.bottle.SodaBottle;
+import edu.jalc.inclass.cocacola.material.label.SodaBottleLabel;
+import edu.jalc.inclass.cocacola.material.lid.SodaBottleLid;
+import edu.jalc.inclass.cocacola.product.liquid.Soda;
+
 public class SodaBottleFillerTest {
 
-	public void testGetInstance(){
-		System.out.println("Testing SodaBottleFiller::getInstance");
-		assert(false);
-	}
+   public void testGetInstance(){
+      System.out.println("Testing SodaBottleFiller::getInstance");
+      SodaBottleFiller sodaBottleFiller = SodaBottleFiller.getInstance();
+      sodaBottleFiller = null;
```

Comment:
A singleton gives only one instance. You have to make sure that the instance returned is the same as another instance returned.